gui-apps/qtgreet-1.0.0.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Aisha Tammy
+# Distributed under the terms of the ISC License
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="QT based greeter for greetd"
+HOMEPAGE="https://gitlab.com/marcusbritanicus/QtGreet"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.com/marcusbritanicus/QtGreet"
+else
+	COMMIT=e425a5acfffc3412c9f5f8184f4ed737846e87d1
+	SRC_URI="https://gitlab.com/marcusbritanicus/QtGreet/-/archive/v${PV}/QtGreet-v${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}"/QtGreet-${COMMIT}
```

Comment:
This folder also needs to be changed, try to see what happens when you emerge this ebuild.
It should fail saying `${S} does not exist` or something similar.

It should probably change to `QtGreet-${PV}` or `QtGreet-v${PV}`, please check which one it is.